.nvmrc

```
@@ -1 +1 @@
-18.14.1
+18.17.1
```

Vercel is also building fine with v20, so I’d suggest we bump this up to the latest too:

```suggestion
20.14.0
```

> I never know what the best approach is, e.g. the minimum supported version, the one from the integration workflows, etc.

I don’t know if there’s a “best practice”, but given that tools that respect `.nvmrc` tend to switch to exactly the version specified, I think it can be OK to upgrade to the latest rather than minimum (it’s not declaring “x or higher” unless you specify a loose version like just `20`, but we’ve had issues in the past with dodgy versions of Node, so I think a fixed version might be safest).

---

This makes total sense, thanks for the Vercel test :thumbsup: